client/components/videos-ui/index.jsx

```diff
@@ -23,6 +26,10 @@ const VideosUi = ( { headerBar, footerBar } ) => {
 		setUserCourseProgression( initialUserCourseProgression );
 	}, [ initialUserCourseProgression ] );
 
+	const [ shouldShowVideoTranslationNotice, setShouldShowVideoTranslationNotice ] = useState(
+		! isEnglish && ! areVideosTranslated
```

Comment:
Hm, since we'd only want to show this message translated, and we can't get strings translated until they're in the codebase, maybe we can commit this with an additional `&& false` here (then remove it when translations are done).

Reply:
Good idea! Updated.